2018-10-20 20:27:33.216 [error] <0.494.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.499.0> exit with reason normal in context child_terminated
2018-10-20 20:27:34.519 [error] emulator Error in process <0.613.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:27:35.528 [error] emulator Error in process <0.629.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:27:36.533 [error] emulator Error in process <0.645.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:27:37.541 [error] emulator Error in process <0.661.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:32:04.027 [error] <0.148.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.153.0> exit with reason normal in context child_terminated
2018-10-20 20:32:04.659 [error] <0.203.0> Supervisor 'Elixir.Tackle.Supervisor' had child 'Elixir.Tackle.Connection' started with 'Elixir.Tackle.Connection':start_link() at <0.204.0> exit with reason killed in context child_terminated
2018-10-20 20:32:05.955 [error] emulator Error in process <0.649.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:05.967 [error] emulator Error in process <0.677.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:06.002 [error] emulator Error in process <0.756.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:06.014 [error] emulator Error in process <0.771.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:06.026 [error] emulator Error in process <0.786.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:06.035 [error] emulator Error in process <0.801.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:07.190 [error] emulator Error in process <0.887.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:07.412 [error] emulator Error in process <0.902.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:12.493 [error] emulator Error in process <0.1000.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:12.702 [error] emulator Error in process <0.1028.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:12.912 [error] emulator Error in process <0.1056.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:13.508 [error] emulator Error in process <0.1071.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:13.709 [error] emulator Error in process <0.1086.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:13.925 [error] emulator Error in process <0.1101.0> with exit value:
{badarith,[{'Elixir.Tackle.RepublishTest.BrokenConsumer',handle_message,1,[{file,"test/integration/republish_test.exs"},{line,17}]},{'Elixir.Tackle.RepublishTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/republish_test.exs"},{line,8}]}]}
2018-10-20 20:32:25.914 [error] <0.1249.0>@rabbit_binary_generator:build_content_frames:95 CRASH REPORT Process <0.1249.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95
2018-10-20 20:32:25.915 [error] <0.1246.0>@rabbit_binary_generator:build_content_frames:95 Supervisor {<0.1246.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.8878>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.1248.0>, {<<"client 127.0.0.1:41486 -> 127.0.0.1:5672">>,1}) at <0.1249.0> exit with reason bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95 in context child_terminated
2018-10-20 20:32:25.915 [error] <0.1246.0> Supervisor {<0.1246.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.8878>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.1248.0>, {<<"client 127.0.0.1:41486 -> 127.0.0.1:5672">>,1}) at <0.1249.0> exit with reason reached_max_restart_intensity in context shutdown
2018-10-20 20:32:25.916 [error] <0.203.0> Supervisor 'Elixir.Tackle.Supervisor' had child 'Elixir.Tackle.Connection' started with 'Elixir.Tackle.Connection':start_link() at <0.339.0> exit with reason killed in context child_terminated
2018-10-20 20:32:25.917 [error] emulator Error in process <0.1253.0> with exit value:
{badarith,[{'Elixir.Tackle.DeliveryHandlerTest','-test delivery consume generates arithmetic exception/1-fun-1-',0,[{file,"test/lib/tackle/delivery_handler_test.exs"},{line,27}]}]}
2018-10-20 20:32:25.917 [error] emulator Error in process <0.1257.0> with exit value:
{{nocatch,{error,12}},[{'Elixir.Tackle.DeliveryHandlerTest','-test delivery consume throws/1-fun-1-',0,[{file,"test/lib/tackle/delivery_handler_test.exs"},{line,39}]}]}
2018-10-20 20:32:25.917 [error] emulator Error in process <0.1259.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"foo">>},[{'Elixir.Tackle.DeliveryHandlerTest','-test delivery consume raises/1-fun-1-',0,[{file,"test/lib/tackle/delivery_handler_test.exs"},{line,33}]}]}
2018-10-20 20:32:26.024 [error] <0.1266.0> Supervisor {<0.1266.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.1267.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,0,infinity,none,[#Fun<amq..>,...],...}) at <0.1268.0> exit with reason killed in context child_terminated
2018-10-20 20:32:26.025 [error] <0.1266.0> Supervisor {<0.1266.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.1267.0>, {amqp_params_network,<<"guest">>,<<"guest">>,<<"/">>,"localhost",5672,0,0,0,infinity,none,[#Fun<amq..>,...],...}) at <0.1268.0> exit with reason reached_max_restart_intensity in context shutdown
2018-10-20 20:32:26.065 [error] emulator Error in process <0.1332.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:26.073 [error] emulator Error in process <0.1347.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:26.083 [error] emulator Error in process <0.1362.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:26.090 [error] emulator Error in process <0.1377.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:27.073 [error] emulator Error in process <0.1592.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:27.084 [error] emulator Error in process <0.1626.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:28.082 [error] emulator Error in process <0.1866.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:28.098 [error] emulator Error in process <0.1881.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:28.121 [error] emulator Error in process <0.1896.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:28.127 [error] emulator Error in process <0.1911.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:28.138 [error] emulator Error in process <0.1926.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:29.089 [error] emulator Error in process <0.2121.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:29.126 [error] emulator Error in process <0.2136.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:29.135 [error] emulator Error in process <0.2151.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:29.138 [error] emulator Error in process <0.2166.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:29.145 [error] emulator Error in process <0.2181.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:30.140 [error] emulator Error in process <0.2376.0> with exit value:
{badarith,[{'Elixir.Tackle.DeadQueueTest.DeadConsumer',handle_message,1,[{file,"test/integration/dead_queue_test.exs"},{line,16}]},{'Elixir.Tackle.DeadQueueTest.DeadConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/dead_queue_test.exs"},{line,7}]}]}
2018-10-20 20:32:42.237 [error] emulator Error in process <0.2591.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:42.243 [error] emulator Error in process <0.2607.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:42.255 [error] emulator Error in process <0.2623.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:42.263 [error] emulator Error in process <0.2639.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:42.270 [error] emulator Error in process <0.2655.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:43.242 [error] emulator Error in process <0.2818.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:43.254 [error] emulator Error in process <0.2844.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:43.261 [error] emulator Error in process <0.2860.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:43.269 [error] emulator Error in process <0.2876.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:44.249 [error] emulator Error in process <0.3068.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:44.261 [error] emulator Error in process <0.3084.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:44.270 [error] emulator Error in process <0.3100.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:44.290 [error] emulator Error in process <0.3116.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:44.302 [error] emulator Error in process <0.3132.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:45.260 [error] emulator Error in process <0.3308.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:45.276 [error] emulator Error in process <0.3324.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:45.290 [error] emulator Error in process <0.3340.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:45.303 [error] emulator Error in process <0.3356.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:45.310 [error] emulator Error in process <0.3372.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:46.287 [error] emulator Error in process <0.3548.0> with exit value:
{badarith,[{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.298 [error] emulator Error in process <0.4703.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.306 [error] emulator Error in process <0.4719.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.316 [error] emulator Error in process <0.4735.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.325 [error] emulator Error in process <0.4751.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.332 [error] emulator Error in process <0.4767.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.341 [error] emulator Error in process <0.4783.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:54.347 [error] emulator Error in process <0.4799.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:55.308 [error] emulator Error in process <0.5004.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:55.316 [error] emulator Error in process <0.5020.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:55.326 [error] emulator Error in process <0.5036.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:55.335 [error] emulator Error in process <0.5052.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:56.317 [error] emulator Error in process <0.5308.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:56.326 [error] emulator Error in process <0.5324.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:56.340 [error] emulator Error in process <0.5340.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:56.351 [error] emulator Error in process <0.5356.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:56.360 [error] emulator Error in process <0.5372.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:57.328 [error] emulator Error in process <0.5612.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:57.345 [error] emulator Error in process <0.5628.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:57.352 [error] emulator Error in process <0.5644.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:57.360 [error] emulator Error in process <0.5660.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:57.372 [error] emulator Error in process <0.5676.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:32:58.371 [error] emulator Error in process <0.5916.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"Raised!">>},[{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerRaisesTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_raises_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.318 [error] emulator Error in process <0.5951.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.326 [error] emulator Error in process <0.5967.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.332 [error] emulator Error in process <0.5983.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.346 [error] emulator Error in process <0.5999.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.352 [error] emulator Error in process <0.6015.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.365 [error] emulator Error in process <0.6031.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:00.372 [error] emulator Error in process <0.6047.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:01.351 [error] emulator Error in process <0.6146.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:01.363 [error] emulator Error in process <0.6172.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:01.372 [error] emulator Error in process <0.6188.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:01.381 [error] emulator Error in process <0.6204.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:02.366 [error] emulator Error in process <0.6364.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:02.382 [error] emulator Error in process <0.6380.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:02.396 [error] emulator Error in process <0.6396.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:02.404 [error] emulator Error in process <0.6412.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:02.415 [error] emulator Error in process <0.6428.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:03.386 [error] emulator Error in process <0.6572.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:03.394 [error] emulator Error in process <0.6588.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:03.404 [error] emulator Error in process <0.6604.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:03.415 [error] emulator Error in process <0.6620.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:04.478 [error] emulator Error in process <0.6780.0> with exit value:
{{nocatch,{1,3,4}},[{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer',handle_message,1,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,19}]},{'Elixir.Tackle.BrokenConsumerThrowsTest.BrokenConsumer','-handle_info/2-fun-0-',3,[{file,"test/integration/broken_consumer_throws_test.exs"},{line,8}]}]}
2018-10-20 20:33:27.697 [error] <0.148.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.153.0> exit with reason normal in context child_terminated
2018-10-20 20:33:27.938 [error] <0.247.0>@rabbit_binary_generator:build_content_frames:95 CRASH REPORT Process <0.247.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95
2018-10-20 20:33:27.938 [error] <0.244.0>@rabbit_binary_generator:build_content_frames:95 Supervisor {<0.244.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.6208>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.246.0>, {<<"client 127.0.0.1:35351 -> 127.0.0.1:5672">>,1}) at <0.247.0> exit with reason bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95 in context child_terminated
2018-10-20 20:33:27.939 [error] <0.244.0> Supervisor {<0.244.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.6208>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.246.0>, {<<"client 127.0.0.1:35351 -> 127.0.0.1:5672">>,1}) at <0.247.0> exit with reason reached_max_restart_intensity in context shutdown
2018-10-20 20:33:30.849 [error] <0.148.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.153.0> exit with reason normal in context child_terminated
2018-10-20 20:33:31.094 [error] <0.247.0>@rabbit_binary_generator:build_content_frames:95 CRASH REPORT Process <0.247.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95
2018-10-20 20:33:31.094 [error] <0.244.0>@rabbit_binary_generator:build_content_frames:95 Supervisor {<0.244.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.6208>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.246.0>, {<<"client 127.0.0.1:55687 -> 127.0.0.1:5672">>,1}) at <0.247.0> exit with reason bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95 in context child_terminated
2018-10-20 20:33:31.094 [error] <0.244.0> Supervisor {<0.244.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.6208>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.246.0>, {<<"client 127.0.0.1:55687 -> 127.0.0.1:5672">>,1}) at <0.247.0> exit with reason reached_max_restart_intensity in context shutdown
2018-10-20 20:33:47.548 [error] <0.148.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.153.0> exit with reason normal in context child_terminated
2018-10-20 20:33:47.754 [error] <0.247.0>@rabbit_binary_generator:build_content_frames:95 CRASH REPORT Process <0.247.0> with 0 neighbours crashed with reason: bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95
2018-10-20 20:33:47.755 [error] <0.244.0>@rabbit_binary_generator:build_content_frames:95 Supervisor {<0.244.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.6208>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.246.0>, {<<"client 127.0.0.1:36870 -> 127.0.0.1:5672">>,1}) at <0.247.0> exit with reason bad argument in call to erlang:size(125) in rabbit_binary_generator:build_content_frames/7 line 95 in context child_terminated
2018-10-20 20:33:47.755 [error] <0.244.0> Supervisor {<0.244.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.6208>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.246.0>, {<<"client 127.0.0.1:36870 -> 127.0.0.1:5672">>,1}) at <0.247.0> exit with reason reached_max_restart_intensity in context shutdown
